apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -88,8 +88,25 @@ interface CommonCalendarEvent extends Event {
     title: string;
 }
 
+export interface Location {
+    /**
+     * @example 'ICS'
+     */
+    building: string;
+
+    /**
+     * @example '174'
+     */
+    room: string;
+
+    /**
+     * If the location only applies on specific days, this is non-null.
+     */
+    days?: string[];
+}
+
 export interface CourseEvent extends CommonCalendarEvent {
-    bldg: string; // E.g., ICS 174, which is actually building + room
+    bldg: Location[];
```

Comment:
Can we rename this to `locations`?

Reply:
looking into this and some of the queries from api also return `bldg` instead of `locations`; going to see if i can also make those changes in this pr or if there's going to be a need for a related backend pr/intermediate formatting

Reply:
update: decided to use intermediate formatting for replacing parts where api gives `bldg` instead of `locations`, a separate migration can be done for backend later if this needs to become standard across all parts of the stack